soccer/gameplay/plays/offense/free_goal.py

```diff
@@ -0,0 +1,50 @@
+import play
+import behavior
+import robocup
+import skills.line_kick
+import tactics.defense
+import main
+import constants
+import enum
+import role_assignment
+
+
+class FreeGoal(play.Play):
```

Comment:
@chachmu should we move this (and the other plays) to a new folder so it dosen't start to clutter our 'real plays?'